audiomentations/augmentations/transforms.py

```diff
@@ -1241,6 +1241,56 @@ def apply(self, samples, sample_rate):
         return distorted_samples
 
     
+class InterruptPulse(BaseWaveformTransform):
+    """
+    Add interrupt pulses to the input audio.
+    """    
+    support_multichannel = False
+    
+    def __init__(
+        self,
+        min_num_interruptions: int = 0,
+        max_num_interruptions: int = 3,
+        noise_level: float = 100.,
+        p: float = 0.5
+    ):
+        """
+        :param min_num_interruptions: Minimum number of interruptions
+        :param max_num_interruptions: Maximum number of interruptions
+        :noise_level: Noise level
+        :param p: The probability of applying this transform
+        """
+        super().__init__(p)
+
+        self.min_num_interruptions = min_num_interruptions
+        self.max_num_interruptions = max_num_interruptions
+        self.noise_level = noise_level
+        if self.min_num_interruptions > self.max_num_interruptions:
+            raise ValueError("min_num_interruptions must not be greater than max_num_interruptions")
+            
+    def randomize_parameters(
+        self, samples: np.array, sample_rate: int = None
+    ):
+        super().randomize_parameters(samples, sample_rate)
+
+        self.parameters["num_interruptions"] = np.random.randint(
+            low=self.min_num_interruptions,
+            high=self.max_num_interruptions
+        )
+        
+    def apply(self, samples: np.array, sample_rate: int = None):
+        max_peak = np.max(samples).astype(int)
```

Comment:
This seems a bit off, as the incoming samples is expected to be audio with float32 samples between -1.0 and 1.0. Unless the audio is maxed out in some places, max_peak and min_peak will be zero.

Did you assume the input to be int16 with max/min values like +/- 32767?

Reply:
Yes, I've assumed so. I've added conversion to int in the beginning, and to float in the end. Or maybe it's better to change code to work with float?

Reply:
In my opinion, it's better to change the code so it works with float directly :) That way, there is no need for the conversion

Reply:
Hmm, OK. I'll try.